contracts/cloud-diff.yml

```diff
@@ -3875,6 +4117,6 @@ components:
 
         For more information and examples, see how to [authenticate with a username and password scheme]({{% INFLUXDB_DOCS_URL %}}/reference/api/influxdb-1x/).
 security:
-  - TokenAuth: []
-  - BasicAuth: []
-  - QuerystringAuth: []
+  - Token authentication: []
```

Comment:
Change back to camelcase to pass swagger validation. Seems kinda pointless since they're arbitrary key names and there's no attribute for a pretty name. I'll look into replacing on docs side.

Reply:
Will it pass validation if you wrap the key in double quotes?

Reply:
> Will it pass validation if you wrap the key in double quotes?

No, it won't. That's what I had previously. Swagger only allows something like [A-Z,a-z,_]